packages/vite/src/node/optimizer/resolve.ts

```diff
@@ -89,22 +90,12 @@ export function expandGlobIds(id: string, config: ResolvedConfig): string[] {
           )
 
           possibleExportPaths.push(
-            ...glob
-              .sync(exportValuePattern, {
-                cwd: pkgData.dir,
-                ignore: ['node_modules'],
-              })
+            ...globSync(exportValuePattern, {
+              cwd: pkgData.dir,
+              expandDirectories: false,
```

Comment:
`tinyglobby` not only aims to replace `fast-glob`, but also `globby`. The latter expands directories (and that's what `tinyglobby` defaults to), the former doesn't, hence we need to pass this option every time.